### ISSUE TITLE
Proposed rewrite of "3.5.2 Publication Conformance" in EPUB A11Y 1.1

### ISSUE THREAD

**Issue by murata2makoto** [mattgarrish/epub-revision#2215]
JDC proposes to replace 

> To indicate that an EPUB Publication conforms to the accessibility requirements of this specification, it MUST include a conformsTo property whose value MUST exactly match (i.e., both in case and spacing) the following pattern:

by 

> Conformance to the accessibility requirements of this specification MUST be represented by a conformTo property exactly matching   (i.e., both in case and spacing) the following pattern.   ...

>Conformance to the accessibility requirements of additional specifications (for example, a specification dedicated to some natural language) MAY be represented by other conformTo property values.

This rewrite is to clearly allow values NOT matching the specified pattern when we would like to indicate that an EPUB Publication conforms to additional specifications (such as those for Japanese EPUB publications).

**Comment by mattgarrish** [comment]
The EPUB spec doesn't restrict the number of dcterms:conformsTo properties you can add, so perhaps we only need to add this to the note that covers a similar issue for using a WCAG string (and maybe look at placement of the example so the notes don't get lost below it):

> NOTE
> 
> The requirement to include a dcterms:conformsTo identifer does prevent EPUB Publications from conforming to other standards, including other accessibility standards and guidelines (e.g., a specification that covers specific natural language issues). An EPUB Publication may have multiple dcterms:conformsTo statements.
> 
> EPUB Creators may also use a conformance URL, as defined in the [Required Components of a Conformance Claim](https://www.w3.org/TR/WCAG/#conformance-required) [[WCAG2](http://localhost:120/a11y/#bib-wcag2)], with the conformsTo property for EPUB Publications that only meet WCAG conformance requirements (i.e., that do not fully conform to this specification).
>
> As [[WCAG2](http://localhost:120/a11y/#bib-wcag2)] does not define how to specify the conformance level in the URL, however, EPUB Creators will have to find an alternative means of relating this information when necessary (e.g., through the accessibility summary).

**Comment by murata2makoto** [comment]
@mattgarrish 

Thank you for the extended note, but "does prevent" should be "does not prevent"

I still think that 

>Conformance to the accessibility requirements of this specification MUST be represented by a conformTo property exactly matching (i.e., both in case and spacing) the following pattern. ...

is better than the current text.

**Comment by mattgarrish** [comment]
> I still think that ... is better than the current text.

We can improve that text, but making the sentence passive makes it unclear what it is requiring or where. It assumes you know what a property is and where it goes.

I'd rephrase it more like this:

> To indicate conformance to the accessibility requirements of this specification, an EPUB Publication's [metadata section](https://www.w3.org/TR/epub-33/#sec-pkg-metadata) [[EPUB-33](http://localhost:120/a11y/#bib-epub-33)] MUST specify a [conformsTo property](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/conformsTo) [[DCTERMS](http://localhost:120/a11y/#bib-dcterms)] exactly matching (i.e., both in case and spacing) the following pattern:

We might also add the package element to the example to make it clearer.
